docs/tech_leadership_principles_guidance.md

```diff
@@ -0,0 +1,68 @@
+
+The harm and erosion of trust that can occur as a lack of embodiment of these principles detrimentally degrades our projects, groups, and communities. 
+
+Where Code of Conduct violations impact one or more individuals directly, lack of adherence to the leadership principles impact the Project, the Group, and the community.
+
+## Expectations for projects and groups
+
+As with all things, [Projects and our groups are Self-governing](https://github.com/cncf/toc/blob/main/PRINCIPLES.md#projects-are-self-governing) towards the governance outcomes defined and expected by the TOC.
+
+It is in this spirit of our operating principles that the TOC expects all projects, groups, and community bodies to self-enforce adherence to the Technical Leadership Principles. We further expect each project, group, and body to respectfully and reasonably address any behavior by an individual that is not in alignment with the Technical Leadership Principles prior to escalating to the TOC for resolution. Among the TOC, escalation is to the TOC Chair and Vice Chair. 
+
+## Escalating to the TOC
+
+In the event that the project or group cannot resolve this conflict with the resources available to them, the project should escalate to the TOC through the private mailing list where the TOC Chair, Vice Chair, or CNCF Staff who will triage.
+
+To escalate to the TOC, please email: cncf-private-toc@lists.cncf.io 
+
+If the escalation concerns a current TOC member, please contact the TOC Chair, Vice Chair, and/or CNCF Staff directly.
+
+Once escalated, two TOC members will be assigned to guide and mentor the project or group and its leadership on methods and options to arrive at an agreeable resolution. Resolutions are crafted to allow for the best outcome for the project or group and its future. The TOC may suggest changes among the leadership roles, responsibilities, and project governance to support that outcome.
+
+## Additional Information and Guidance
+
+### Conflict is likely
+
+Innovation is often born from the diversity of opinion, perspective, and experience of individuals collaborating towards a common goal or objective. Fostering an engaging and exploratory discussion, debate, and exchange of ideas to drive innovation requires a foundation of respect, grace, and curiosity so all voices may be elevated, heard, and incorporated into the discussion.
+
+Any personal, social views and political opinions exposed to the community can destroy and undermine trust of the leader holding them because those personal views may not align with CNCF values and may be seen to promote intolerance. Individuals have a right to their personal, private perspectives, but as leaders in our community we are expected to embody and practice the technical leadership principles and uphold our Code of Conduct through our behavior and engagements, serving as the arbiter of equity and fairness within projects, within the community, within CNCF spaces, and at CNCF events.
```

Comment:
This paragraph contains information that's also in the next one (about trust). Maybe move and merge? I like the subsections. (minor comment, just smth i noticed)